Scripts/SetIfEmpty/SetIfEmpty_test.py

```diff
@@ -0,0 +1,52 @@
+
+from SetIfEmpty import getValueToSet
+
+def test_WhenValueIsAValidString_ShouldReturnValue():
```

Comment:
please change to pep8 convention

Reply:
Regarding the test names? 
Couldn't find anything regarding unit test naming in this guide:
https://www.python.org/dev/peps/pep-0008/

Looking at other files, did you mean naming the tests (for example) 
test_WhenValueIsEmptyDictionary_ShouldReturnDefaultValue  -->
test_value_is_empty_dictionary ?